raiden-dapp/src/components/ChannelDepositDialog.vue

```diff
@@ -2,7 +2,7 @@
   <raiden-dialog
     class="channel-deposit"
     :visible="visible"
-    :fullscreen="true"
+    :fullscreen="false"
```

Comment:
Have you checked if that looks as intended? There can be weird behavior if there is a parent element in the DOM with `position: relative;`